lib/dry/system/container.rb

```diff
@@ -667,28 +633,41 @@ def inherited(klass)
           super
         end
 
-        private
+        protected
```

Comment:
Look, it's a rare sighting of `protected` in its native habitat! I'm using this here because in `.load_imported_component` we need to call `.load_component` on the container that we're importing, which will be another `Dry::System::Container` instance, aka the exact reason `protected` exists :)